### ISSUE TITLE
function to write Root.in

### ISSUE THREAD

**Issue by schwemro** [phydrus/phydrus#35]
Are there any plans to provide a function which writes the input file for root growth?

**Comment by raoulcollenteur** [comment]
Hi @schwemro ! Not sure what you mean, I couldn't find a reference to ROOT.IN in the Hydrus 4.08 manual? What process is that? We do already support root growth through the SELECTOR.IN (Block D), added through the method:

`Model.add_root_growth()`

Is that what you're looking for?

Cheers,
Raoul

**Comment by schwemro** [comment]
Yes, you support it, but there is no output on rooting depth (e.g. root depth at time step t).
